src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.lib.Dao;
+import mate.academy.model.CinemaHall;
+
+@Dao
```

Comment:
where should you put this annotation?

Reply:
We need this annotation in implementations only